README.md

```diff
@@ -15,6 +15,13 @@ You may also be interested in the hosted TensorBoard solution at
 share your ML experiments for free. For example, [this experiment][] shows a
 working example featuring the scalar dashboard.
 
+TensorBoard can run entirely offline on your local machine, in a corporate
+environment, or in a datacenter, without connecting to Google servers. There is
+one exception: the optional profile plugin, when active, connects to a
+Google-hosted CDN to load the Google Charts library. Your data is never shared
+with Google unless you explicitly upload it to [TensorBoard.dev][] for public
```

Comment:
I suggest moving anything regarding TensorBoard.dev to a separate paragraph or section. We should avoid even the hint of conflation of TensorBoard as an open source project with the hosted service.